app/controllers/application_controller.rb

```diff
@@ -66,6 +66,7 @@ def ensure_mentor!
   def ensure_supermentor!
```

Comment:
We have a similar method named `ensure_mentor!`, do we also want to add the `staff` check there?